js/ui/tree_list/ui.tree_list.rows.js

```diff
@@ -29,36 +29,37 @@ exports.RowsView = rowsViewModule.views.rowsView.inherit((function() {
         return $iconElement;
     };
 
-    const renderIcons = function($container, row) {
-        const level = row.level;
-
-        for(let i = 0; i <= level; i++) {
-            $container.append(createIcon(i === level && row.node.hasChildren, row.isExpanded));
-        }
-    };
-
     return {
-        _renderExpandIcon: function($container, options) {
+        _renderIconContainer: function($container, options) {
             const $iconContainer = $('<div>')
                 .addClass(TREELIST_EXPAND_ICON_CONTAINER_CLASS)
                 .appendTo($container);
 
-            renderIcons($iconContainer, options.row);
-
             options.watch && options.watch(function() {
```

Comment:
If we don't have 'options.watch' do we need to render icons by 'this._renderIcons($iconContainer, options)' method?
As it was here: https://github.com/DevExpress/DevExtreme/pull/11347/files#diff-810751d755fc6bec8c41133709b7e5acL46